src/styles.css

```diff
@@ -0,0 +1,601 @@
+}
+.Search-Below {
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  width: 80vw;
+  height: 2.1rem;
+  margin: auto;
+  display: flex;
+  margin-bottom: 10px;
+}
+.Search > input,
+.Search-Below > input {
+  padding-left: 5px;
+  width: 100%;
+  height: 1.8rem;
+  border: none;
+  outline: none;
+}
+.Search > button,
+.Search-Below > button {
+  background-color: transparent;
+  border: none;
+  outline: none;
+  margin-top: 4px;
+}
+.Cardbox {
+  margin: auto;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: flex-start;
+  box-shadow: 0 0 6px 0px black;
+  padding: 1rem;
+  margin-top: 1rem;
+}
+.cartContent {
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+}
+.incdecbtn {
+  padding: 1px 8px;
+  color: purple;
+  font-size: 22px;
+  border-radius: 50%;
+  border: none;
+  outline: none;
+  background-color: rgb(221 219 219);
+}
+.cartOrder {
+  margin: auto;
+  border: 1px solid rgba(0, 0, 0, 0.5);
+  margin-top: 1rem;
+  padding: 1.5rem;
+  text-align: start;
+}
+.cartdetails {
+  display: flex;
+  justify-content: space-between;
+  margin: 5px 0px;
+}
+.soldout {
+  transform: translate(29px, -172px) rotate(-45deg);
+  font-size: 3rem;
+  color: var(--danger);
+  position: absolute;
+}
+.productbox {
+  display: grid;
+  grid-template-columns: 1fr 5fr;
+}
+.filterboxDesk {
+  border: 1px solid rgba(0, 0, 0, 0.5);
+  border-top: none;
+  padding: 8px;
+  height: 100vh;
+  position: sticky;
+  text-align: start;
+  top: 0;
+}
+.filterboxDesk > label {
+  display: block;
+}
+.filterboxMobile {
+  display: none;
+}
+.filterboxMobile_buttons {
+  display: none;
+  position: fixed;
+  bottom: 0;
+  z-index: 2;
+  background-color: white;
+}
+.div_filters {
+  font-size: 20px;
+  padding: 10px;
+  width: 44vw;
+  border: 1px solid rgba(0, 0, 0, 0.5);
+}
+.sortBox,
+.filterBox {
+  border: 1px solid rgba(0, 0, 0, 0.5);
+  width: 95vw;
+  background-color: white;
+  position: fixed;
+  text-align: start;
+  padding-left: 1rem;
+  bottom: 0;
+  z-index: 2;
+}
+.filterBox > label {
+  display: block;
+  margin: 1rem 0;
+}
+.sortBox > p {
+  margin-bottom: 1rem;
+  cursor: default;
+}
+.closeButton_filter_sort {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 1rem;
+}
+.closeButton_filter_sort > button {
+  padding: 5px;
+  font-size: 18px;
+  font-weight: bold;
+  border: none;
+  background-color: white;
+  outline: none;
+  color: purple;
+}
+.dark_background {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  right: 0;
+  z-index: 1;
+  left: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+.home-image {
+  width: 100%;
+}
+.home-card {
+  cursor: pointer;
+  position: relative;
+  border: 1px solid rgba(0, 0, 0, 0.5);
+  margin: 1rem;
+}
+.home-card:hover {
+  transform: scale(1.1);
+  transition: all 0.3s;
+}
+.image-size {
+  width: 240px;
+  height: 240px;
+}
+.text-overlay-position {
+  position: absolute;
+  top: 0;
```

Comment:
use [inset](https://developer.mozilla.org/en-US/docs/Web/CSS/inset). Its shorthand for top, right, botoom, left.